.github/workflows/bazel.yml

```diff
@@ -44,7 +56,10 @@ jobs:
 
             - name: "Report test coverage"
               run: |
-                  tools/bazelisk coverage --combined_report=lcov --instrument_test_targets //...
+                  tools/bazelisk coverage --combined_report=lcov --instrument_test_targets --test_verbose_timeout_warnings //...
+
+            - name: Setup upterm session
+              uses: lhotari/action-upterm@v1
```

Comment:
This can come in handy, but we'll want to remove it otherwise tests won't terminate by themselves, no?

Reply:
I'm not done with the PR yet, upterm's there to debug for now − we should indeed remove it when we're done. For some reason the MacOS job fails at `Submit report to Coveralls`: [here](https://github.com/boragokbakan/mpacklog/actions/runs/4744791127/jobs/8426188109)

Based on logs I suspect the previous step is not creating the coverage report properly on MacOS, and neither can I connect to the macOS upterm session for some reason 😖.

Reply:
We're all green now then! :)